app/commands/track/determine_concept_map_layout.rb

```diff
@@ -19,6 +19,8 @@ def call
         levels: concept_levels,
         connections: concept_connections
       }
+    rescue TrackHasCyclicPrerequisiteError
+      Github::Issue::OpenForDependencyCycle.(track)
```

Comment:
This should needs wrapping as a job, as this is happening in-progress for the user.

I think we should also then re-raise the exception (you can use `raise` with no args), and catch and handle it in the UI.